### DISCUSSION TITLE
API Key

### DISCUSSION THREAD

**Discussion by jacksonsophat** [pocketbase/pocketbase#2241] [Q&A]
Does PocketBase have an Api key or something like to make sure that only a certain website can make a API call?

Thank you

**Comment by ganigeorgiev** [comment]
There is no built-in API keys functionality.

Your question doesn't have enough details to understand what is your use case and why do you want to prevent other websites from making API calls, but if you want to restrict other websites to make direct browser requests to your PocketBase server you can define a CORS origin with the `--origins=example.com` flag.

Alternatively you can specify your own random string to treat it as API key and check it in the collections API rules via the `@request.query.*` or `@request.headers.*` filters. You can find more details about the `@request.*` filters in https://pocketbase.io/docs/api-rules-and-filters/#filters-syntax.
